docs/deployment-process/issue-tracking/azure-devops.md

```diff
@@ -0,0 +1,90 @@
+Octopus does this by using the build information associated with any packages contained in the release to request work item references from Azure DevOps.
+
+To supply the build information:
+
+1. Install one of our official build server plugins with support for our metadata step. This list currently includes [JetBrains TeamCity](https://plugins.jetbrains.com/plugin/9038-octopus-deploy-integration), [Atlassian Bamboo](https://marketplace.atlassian.com/apps/1217235/octopus-deploy-bamboo-add-on?hosting=server&tab=overview), and [Azure DevOps](https://marketplace.visualstudio.com/items?itemName=octopusdeploy.octopus-deploy-build-release-tasks). We're currently working on adding support for the Jenkins plugin.
+
+2. Update your build process to add and configure the Octopus Metadata step.
+
+3. Ensure your Octopus Metadata step has set the 'Process commit messages' or 'Work Items Source' field.
+
+## Connect Octopus to Azure DevOps {#connect-octopus-to-azure-devops}
+
+1. Configure the Azure DevOps Issue Tracker extension in Octopus Deploy.
+
+    In the Octopus web portal, navigate to **{{Configuration,Settings,Azure DevOps Issue Tracker}}** and set the following values:
+
+    - **Azure DevOps Base URL**. This tells Octopus where the Azure DevOps instance is located.
+
+    - Unless the Azure DevOps instance is public, you'll need to supply a **Personal Access Token**, created in the Azure DevOps User Settings (under Personal access tokens), with authorization to read scopes **Build** and **Work items**.
+
+    - Ensure the **Is Enabled** property is set as well.
+
+2. Configure the Release Note Options _(optional)_.
+
+    - **Release Note Prefix**: If specified, Octopus will look for a work item comment that starts with the given prefix text and use whatever text appears after the prefix as the release note, which will be available in the [build information](/docs/packaging-applications/build-servers/index.md#build-information) as the work item's description. If no comment is found with the prefix then Octopus will default back to using the title for that work item.
+
+    For example, a prefix of `Release note:` can be used to identify a customer friendly work item title vs a technical feature or bug fix title.
+
+When configured, this integration will retrieve Azure DevOps work item details and add details to your releases and deployments.
+
+## Troubleshooting
+
+### Error message: "Unable to obtain some work item data."
+
+The rest of the message will have more information about what went wrong, but this typically indicates a failure to do one of the following:
+
+* contact Azure DevOps over the network
+* authenticate using the specified Personal Access Token to read scopes **Build** and **Work items**
+* retrieve build and work item data, and work item comments
+* interpret the data
+
+If you can't resolve the problem and want to create a release, some options are:
```

Comment:
Do we want to include this information? i.e. Turn it off? 😬

Reply:
Well, we don't want customers to accidentally create releases if these linkages are needing to be repaired. But we also don't want them to be unable to release if they can't repair.

Reply:
Should I keep it simple with one sentence? Which would be like this -
```markdown
If you can't resolve the problem and want to create a release, disable the Azure DevOps Issue Tracker extension under **{{Configuration,Settings,Azure DevOps Issue Tracker}}**.
```

Reply:
I can just remove the bit about not pushing metadata, since they probably have if they've hit this this point.

Reply:
You could add to line 75, `if you ever need to disable the Azure DevOps Issue Tracker extension you can do so under **{{Configuration,Settings,Azure DevOps Issue Tracker}}**.`

Reply:
That avoids explicitly saying turn it off, but it provides the information if they need it.

Reply:
Will do.